### PR TITLE
feat: add input hover and active backgroundColor

### PR DESCRIPTION
**Pull request by Pan-yongyong**
https://github.com/ant-design/ant-design/issues/44591

### CODE REVIEW

File: components/input/style/index.ts [hunk 1]
```diff
@@ -1048,6 +1059,8 @@ export const initComponentToken = (token: GlobalToken): SharedComponentToken =>
     activeShadow: `0 0 0 ${controlOutlineWidth}px ${controlOutline}`,
     errorActiveShadow: `0 0 0 ${controlOutlineWidth}px ${colorErrorOutline}`,
     warningActiveShadow: `0 0 0 ${controlOutlineWidth}px ${colorWarningOutline}`,
+    hoverColorBgContainer: '#FFF',
+    activeColorBgContainer: '#FFF',
```
Comment (afc163) [inline components/input/style/index.ts:1063]:
```suggestion
    hoverColorBgContainer: 'transparent',
    activeColorBgContainer: 'transparent',
```

File: components/input/style/index.ts [hunk 2]
```diff
@@ -66,6 +66,16 @@ export interface SharedComponentToken {
...
+  /**
+   * @desc 输入框hover状态时背景颜色
+   * @descEN #FFFFFF
+   */
+  hoverColorBgContainer: string;
+  /**
+   * @desc 输入框激活状态时背景颜色
+   * @descEN #FFFFFF
+   */
+  activeColorBgContainer: string;
```
Comment (afc163) [inline components/input/style/index.ts:78]: activeBg

**Comment by afc163** [comment]
新特性，发到 feature 分支吧

File: components/input/style/index.ts [hunk 3]
```diff
@@ -66,6 +66,16 @@ export interface SharedComponentToken {
    * @descEN Box-shadow when active in warning status
    */
   warningActiveShadow: string;
+  /**
+   * @desc 输入框hover状态时背景颜色
+   * @descEN Background color when the input box hovers
+   */
+  hoverColorBgContainer: string;
```
Comment (MadCcc) [inline components/input/style/index.ts:73]:
```suggestion
  hoverBg: string;
```

File: components/input/style/index.ts [hunk 4]
```diff
@@ -66,6 +66,16 @@ export interface SharedComponentToken {
...
+  /**
+   * @desc 输入框激活状态时背景颜色
+   * @descEN Background color when the input box is activated
+   */
+  activeColorBgContainer: string;
```
Comment (MadCcc) [inline components/input/style/index.ts:78]:
```suggestion
  activeBg: string;
```

**Pull request closed without merging**